pages/index.jsx

```diff
@@ -1,13 +1,16 @@
-import { useEffect, useState } from 'react';
+import { useEffect, useState, useRef } from 'react';
 import Head from 'next/head';
 import Link from 'next/link';
 
 import apiService from '../services/apiService';
 import LoadingStatus, { loadingStates } from '../components/LoadingStatus';
+import SkipToContent from '../components/SkipToContent';
 
 export default function Home() {
   const [teamMembers, setTeamMembers] = useState([]);
   const [loadingStatus, setLoadingStatus] = useState(loadingStates.EMPTY);
+  const contentRef = useRef(null);
+  const skipOnClick = () => contentRef.current.focus();
```

Comment:
I am wondering if using `useRef` and the DOM method `.focus()` is necessary. I was having trouble getting focus to switch without it but I also initially had a setting turned on in my browser that affected tabbing through a page.